docs/content/preview/yugabyte-voyager/reference/bulk-data-load/import-data-file.md

```diff
@@ -0,0 +1,298 @@
+| --target-db-name <name> | Target database name. |
+| --target-db-password <password>| Target database password. Alternatively, you can also specify the password by setting the environment variable `TARGET_DB_PASSWORD`. If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes. |
+| --target-db-port <port> | Port number of the target database machine. (default: 5433) |
+| --target-db-schema <schemaName> | Schema name of the target database. MySQL and Oracle migrations only. |
+| --target-db-user <username> | Username of the target database. |
+| [--target-ssl-cert](../../yb-voyager-cli/#ssl-connectivity) <certificateName> | Name of the certificate which is part of the SSL `<cert,key>` pair. |
+| [--target-ssl-key](../../yb-voyager-cli/#ssl-connectivity) <keyName> | Name of the key which is part of the SSL `<cert,key>` pair. |
+| [--target-ssl-crl](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing the SSL certificate revocation list (CRL).|
+| [--target-ssl-mode](../../yb-voyager-cli/#ssl-connectivity) <SSLmode> | One of `disable`, `allow`, `prefer`(default), `require`, `verify-ca`, or `verify-full`. |
+| [--target-ssl-root-cert](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing SSL certificate authority (CA) certificate(s). |
+| --verbose | Display extra information in the output. (default: false) |
+| -y, --yes  | Answer yes to all prompts during the export schema operation. |
+
+<!-- To do : document the following arguments with description
+| --continue-on-error |
+| --enable-upsert |
+| --target-endpoints |
+| --use-public-ip | -->
+
+## Examples
+
+The following examples use the `--data-dir` argument along with `--file-table-map` argument and contextually differ based on whether the data is imported from local disk or cloud storage options.
+
+{{< tabpane text=true >}}
+
+ {{% tab header="CSV" %}}
+
+Import data for CSV files using `import data file` command by providing the argument `--format 'csv'` in the command.
+
+### Import data file from local disk
+
+The `--data-dir` argument is a path to the local directory where all the CSV files are present, and the `--file-table-map` argument provides a comma-separated mapping between each CSV file in `--data-dir` to the corresponding table in the database (you can mention case-sensitive table names as well); each file has a header, delimiter as ',' (default), escape and quote character as '"'(default).
+
```

Comment:
@priyanshi-yb - you can mention case-sensitive table names as well, refer second example on how to pass- for this can we just include an example instead. and the example would be ??

Reply:
example- `--file-table-map 'orders.csv:"Orders",products.csv:"Products"'`

Reply:
example - `--file-table-map 'foo.csv:"Foo"'`